dbms/src/Flash/Mpp/AsyncRequestHandler.h

```diff
@@ -137,6 +137,7 @@ class AsyncRequestHandler : public AsyncRequestHandlerBase
     {
         std::unique_lock<std::mutex> ul(is_close_conn_called_mu);
         condition_cv.wait(ul, [this]() { return is_close_conn_called; });
+        ul.unlock();
```

Comment:
the destructor of `ul` will unlock it, so why call unlock explicitly here?

Reply:
> the destructor of `ul` will unlock it, so why call unlock explicitly here?

oh, I ignore the destructor, and i will delete it.